lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -1488,7 +1488,7 @@ define(['backbone', 'jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers
 
                         topicsJson = {
                             course_wide_discussions: {
-                                children: ['Topic_C_1'],
+                                children: [['Topic_C_1', 'entry']],
```

Comment:
I was thinking about the Python tests. 

@andy-armstrong or @bjacobel would be better at suggesting the appropriate place for constants in JavaScript.